.ci/deps.sh

```diff
@@ -175,3 +175,12 @@ fi
 echo '' > dummy
 coala-ci --bears CheckstyleBear --files dummy --no-config --bear-dirs bears || true
 coala-ci --bears ScalaLintBear --files dummy --no-config --bear-dirs bears || true
+
+if [ ! -e ~/elm_format/elm-format ]; then
+  mkdir -p ~/elm_format
+  export PATH=$PATH:~/elm_format
+  wget -nc -O elm-format.tgz https://github.com/avh4/elm-format/releases/download/0.5.2-alpha/elm-format-0.17-0.5.2-alpha-linux-x64.tgz
+  tar -xvzf elm-format.tgz
+  mv elm-format ~/elm_format
```

Comment:
I have to work on them, main focus was to get all green ticks. Marking WIP. :D Thanks a lot for your meaningful reviews.